### ISSUE TITLE
-f option does not work at all?

### ISSUE THREAD

**Issue by pgilmorepf** [Microsoft/dts-gen#30]
Steps:

* npm install -g dts-gen
* (Tried with existing .js files, and with a new empty folder)
* create test.js with the following contents:
  ```
  var x = {};
  function TheFunction(y) {
  }
  TheFunction(x);
  ```
* dts-gen -f test.js
* fails with the following error:
  ```    T:\depot\sdks\JavaScriptSDK\PlayFabSDK>dts-gen -f test.js
    Unexpected crash! Please log a bug with the commandline you specified.
    C:\Users\MY USERNAME\AppData\Roaming\npm\node_modules\dts-gen\bin\lib\run.js:109
            throw e;
            ^

    Error
        at ArgsError.Error (native)
        at new ArgsError (C:\Users\MY USERNAME\AppData\Roaming\npm\node_modules\dts-gen\bin\lib\run.js:25:28)
        at Object.<anonymous> (C:\Users\MY USERNAME\AppData\Roaming\npm\node_modules\dts-gen\bin\lib\run.js:39:15)
        at Module._compile (module.js:409:26)
        at Object.Module._extensions..js (module.js:416:10)
        at Module.load (module.js:343:32)
        at Function.Module._load (module.js:300:12)
        at Function.Module.runMain (module.js:441:10)
        at startup (node.js:139:18)
        at node.js:974:3
  ```

**Comment by pgilmorepf** [comment]
I actually can't get anything BUT this error, regardless of command line args or other files.

**Comment by andy-ms** [comment]
Fixed by #35.

**Comment by tischsoic** [comment]
I have version number 0.5.2 and I'm getting the same error as above. What am I doing wrong? How can I generate d.ts file for js file which is not npm module?
